packages/template-retail-react-app/config/default.js

```diff
@@ -15,6 +15,11 @@ module.exports = {
             showDefaults: true,
             interpretPlusSignAsSpace: false
         },
+        login: {
+            passwordless: {
+                callbackURI: 'https://webhook.site/27761b71-50c1-4097-a600-21a3b89a546c'
+            }
+        },
```

Comment:
Example of how the passwordless login config would look like if mode is `callback` !

Reply:
I know that I've seen some configuration settings from Jinsu on the stuff that he was working on (social login) I think. Is there a place that I can see how those configurations co-exist so I can get a bigger picture of whether or not our chosen configuration works as a whole. 

I'm assuming that they all live under the umbrella of login in the PWA config? But it looks like maybe they get split up when applied to the provider?

Reply:
Yes! I created a doc containing the config file changes for both social and passwordless login [here](https://salesforce.quip.com/t1VfAbRqsbmT).